<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>550</width>
    <height>400</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(33, 33, 33)</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>60</y>
     <width>121</width>
     <height>16</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(20, 255, 236)</string>
   </property>
   <property name="text">
    <string>Выберите базу данных</string>
   </property>
  </widget>
  <widget class="QSplitter" name="splitter">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>110</y>
     <width>131</width>
     <height>121</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Vertical</enum>
   </property>
   <widget class="QPushButton" name="pushButton">
    <property name="styleSheet">
     <string notr="true">color: rgb(20, 255, 236)</string>
    </property>
    <property name="text">
     <string>services</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_2">
    <property name="styleSheet">
     <string notr="true">color: rgb(20, 255, 236)</string>
    </property>
    <property name="text">
     <string>workers</string>
    </property>
   </widget>
  </widget>
  <widget class="QSplitter" name="splitter_2">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>110</y>
     <width>131</width>
     <height>121</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Vertical</enum>
   </property>
   <widget class="QPushButton" name="pushButton_3">
    <property name="styleSheet">
     <string notr="true">color: rgb(20, 255, 236)</string>
    </property>
    <property name="text">
     <string>client</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_4">
    <property name="styleSheet">
     <string notr="true">color: rgb(20, 255, 236)</string>
    </property>
    <property name="text">
     <string>zakazi</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
